src/modal/modal.scss

```diff
@@ -28,6 +28,7 @@
     z-index: 1000;
     max-width: 50%;
     min-width: 400px;
+    max-height: 100%;
```

Comment:
Three problems:
- modal uses content box, so this 100% does not include padding
- for portrait orientation, this will not consider the notch, though that's an extremely rare case I guess
- scrolling is not enabled

Reply:
I'm going to close this PR and we can revisit another time.